README.md

```diff
@@ -1,2 +1,220 @@
+/) [![Stackoverflow](https://img.shields.io/badge/Ask%20for%20help%20on-Stackoverflow-orange)](https://stackoverflow.com/questions/tagged/wso2is)
+[![Join the chat at https://join.slack.com/t/wso2is/shared_invite/enQtNzk0MTI1OTg5NjM1LTllODZiMTYzMmY0YzljYjdhZGExZWVkZDUxOWVjZDJkZGIzNTE1NDllYWFhM2MyOGFjMDlkYzJjODJhOWQ4YjE](https://img.shields.io/badge/Join%20us%20on-Slack-%23e01563.svg)](https://join.slack.com/t/wso2is/shared_invite/enQtNzk0MTI1OTg5NjM1LTllODZiMTYzMmY0YzljYjdhZGExZWVkZDUxOWVjZDJkZGIzNTE1NDllYWFhM2MyOGFjMDlkYzJjODJhOWQ4YjE)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/wso2/product-is/blob/master/LICENSE)
+[![Twitter](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=wso2)
+---
+
+The Asgardio Tomcat OIDC Agent enables you to add OIDC-based login, logout to your Apache Tomcat web apps with
+ minimum hassle.
+
+
+- [Getting Started](#getting-started)
+- [How it works](#how-it-works)
+- [Integrating Asgardio Tomcat OIDC Agent](#integrating-asgardio-tomcat-oidc-agent)
+  * [To your existing webapp](#to-your-existing-webapp)
+  * [To your Java source project](#to-your-java-source-project)
+- [Building from the source](#building-from-the-source)
+- [Contributing](#contributing)
+  * [Reporting issues](#reporting-issues)
+- [License](#license)
+
+## Getting started
+
+You can experience the capabilities of Asgardio Tomcat OIDC Agent by following this small guide which contains main
+ sections listed below.
+
+  * [Prerequisites](#prerequisites)
+  * [Configuring the sample](#configuring-the-sample)
+  * [Configuring Identity Server](#configuring-identity-server)
+  * [Running the sample](#running-the-sample)
+
+### Prerequisites
+1. WSO2 Identity Server and it's [prerequisites](https://is.docs.wso2.com/en/next/setup/installing-the-product/).
+
+A sample app for demonstrating OIDC based authentication/authorization, logout and attribute retrieval is hosted at:
+[TODO link]
+
+You can download the pre-built oidc-sample-app.war from [TODO link]
+
+### Configuring the sample
+1.  Add the following entry to the `/etc/hosts` file of your machine to configure the hostname.
+   ```
+   127.0.0.1 localhost.com
+   ```
+
+### Configuring Identity Server
+
+Here we are using WSO2 Identity Server as the OpenID Provider. The sample can be configured with any other preferred
+ OpenID Provider as well.
+ 
+1. Start the WSO2 IS. 
+2. Access WSO2 IS management console and create a service provider (ex:- oidc-sample-app)
+   
+   For the service provider, configure Oauth/OpenID Connect under Inbound Authentication Configuration. In this
+    configuration,
+   use following parameters and options,
+     
+       Callback URL - http://localhost:8080/oidc-sample-app/oauth2client
```

Comment:
localhost.com

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-oidc-agent/pull/7/commits/bf4f7fa7367a16bd78c9cbabc954df5fb7451e18